assets/css/styleguide.scss

```diff
@@ -442,24 +444,28 @@ $site-top:           124px;
   }
 }
 
-.usa-grid-example-blank {
-  text-align: center;
+.grid-example-blank {
+  .usa-grid {
+    text-align: center;
+  }
 }
 
-.usa-grid-text {
-  background: $color-grid-dark;
+.grid-text {
+  .usa-grid {
+    background: $color-grid-dark;
 
-  > * {
-  background: $color-grid-light;
-  color: #000;
-  }
+    > * {
```

Comment:
This is hard to see in gh, but shouldn't the lines below be indented in one more?